themes/bootstrap3/templates/RecordDriver/DefaultRecord/result-list.phtml

```diff
@@ -20,19 +20,21 @@
   <?php endif ?>
   <div class="media-body">
     <div class="result-body">
-      <div>
-        <a href="<?=$this->escapeHtmlAttr($recordLinker->getUrl($this->driver))?>" class="title getFull" data-view="<?=isset($this->params) ? $this->params->getOptions()->getListViewOption() : 'list' ?>">
-          <?=$this->record($this->driver)->getTitleHtml()?>
-        </a>
-        <?php foreach ($this->driver->tryMethod('getTitlesAltScript', [], []) as $altTitle): ?>
-          <div class="title-alt">
-            <?=$this->escapeHtml($altTitle)?>
-          </div>
-        <?php endforeach; ?>
-      </div>
+      <header>
+        <h2>
+          <a href="<?=$this->escapeHtmlAttr($recordLinker->getUrl($this->driver))?>" class="title getFull" data-view="<?=isset($this->params) ? $this->params->getOptions()->getListViewOption() : 'list' ?>">
+            <?=$this->record($this->driver)->getTitleHtml()?>
+          </a>
+          <?php foreach ($this->driver->tryMethod('getTitlesAltScript', [], []) as $altTitle): ?>
+            <div class="title-alt">
+              <?=$this->escapeHtml($altTitle)?>
+            </div>
+          <?php endforeach; ?>
+        </h2>
 
-      <div>
         <?php if ($this->driver->isCollection()): ?>
+          </header>
```

Comment:
This close tag is inside a condition, but there's also a similar close tag AFTER the end of the if statement. I don't think this is correct, but maybe I'm misreading something.

Reply:
I think that closing tag belongs to another `if`. The closing `header` is inside the `if` and the `else` condition, like:
```
<?php if ($this->driver->isCollection()): ?>
  </header>
  <div>
<?php else: ?>
  <?php ... if (!empty($summAuthors)): ?>
    <?php foreach ($summAuthors as $i => $summAuthor): ?>
      ...
    <?php endforeach; ?>
  <?php endif; ?>
  </header>
  <div>
```

Reply:
Sorry, I see that you're right! However, I wonder if it would be more readable and easier to understand/maintain if we eliminated the duplicate close tag by refactoring so it's more like:

```
if (!$this->driver->isCollection()): ?>
  <?php ... if (!empty($summAuthors)): ?>
  ...
  <?php endif; ?>
<?php endif; ?>
</header>
<div>

<?php if ($this->driver->isCollection()): ?>
  ...
<?php else: ?>
  ...
<?php endif; ?>
```

Do you see what I mean? It adds an additional if statement to the template, but it makes the overall flow of the logic a little easier to follow.

Reply:
Yes, I agree, that would be easier to read it! 
I will adjust it.

Reply:
Sorry, but it is not possible to eliminated the duplicate close tag `</header>`. 
The point is, that we can't move the closing tag outside of that condition:
`<?php if (!$this->driver->isCollection() ?>`
The closing tag has to be after `$summAuthors` but there are following $journalTitle, $summDate abd other data which is rendered in the same condition.
We would need something like this, using the same if-query twice:
if (!$this->driver->isCollection()): ?>
  <?php ... if (!empty($summAuthors)): ?>
  ...
  <?php endif; ?>
<?php endif; ?>
</header>
<div>
if (!$this->driver->isCollection()): ?>
<?php if (!empty($journalTitle)): ?>
...
<?php elseif (!empty($summDate)): ?>
...
<?php endif; ?>

<?php if ($this->driver->isCollection()): ?>
  ...
<?php endif; ?>

Reply:
Sure, I can do that. Meaning that `else` (from "keeps the rest of the existing if..else")  will be the same case like `<?php if (!$this->driver->isCollection()): ?> `
I will change it right now.

Reply:
Yes, exactly! Thanks!